src/services/tx/txSender.ts

```diff
@@ -288,3 +290,24 @@ export const dispatchBatchExecution = async (
 
   return result.hash
 }
+
+export const dispatchSpendingLimitTxExecution = (
```

Comment:
Naming-wise, we use `dispatch` to indicate the function is also doing an even dispatch.

I guess we should add some custom tx events here, as per the standup discussion.

Reply:
Exactly, I was extracting it now so we can easily dispatch the events in this function but I would do it in a separate PR to keep the scope smaller.